assets/js/base/components/product-grid/container.js

```diff
@@ -3,48 +3,42 @@
  */
 import { Component } from 'react';
 import PropTypes from 'prop-types';
-import { addQueryArgs, getQueryArg } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
 import ProductGrid from './index';
-import withBrowserHistory from '@woocommerce/base-hocs/with-browser-history';
-import withBrowserLocation from '@woocommerce/base-hocs/with-browser-location';
+import withQueryStringValues from '@woocommerce/base-hocs/with-query-string-values';
 
 class ProductGridContainer extends Component {
-	state = {
-		currentPage: this.props.currentPage || 1,
-		sortValue: this.props.sortValue,
-	};
-
 	onPageChange = ( newPage ) => {
-		this.setState( {
-			currentPage: newPage,
+		const { updateQueryStringValues } = this.props;
+		updateQueryStringValues( {
```

Comment:
probably could just use `this.props.updateQueryStringValues` directly here since it's only used once in this method.  There's a few other places in this file where the same comment applies.

Reply:
Done: 7d0e9589be085bf18d86dd862bd2ef1174a99916.